### PR TITLE
Recipe app

### PR DESCRIPTION
**Pull request by Mhdez221993**
## Project requirements

- You should follow the layout of the wireframes provided. You should personalize the rest of the design including colors, typographies, spacings, etc.
- Login page and registration page:
    - Should be built with Devise.
- Food list:
    - Should display a list of food as in the wireframe.
    - Should lead to the form that allows users to add new food.
- Recipies list:
    - Should display a list of recipes created by the logged-in user as in the wireframe.
    - Should lead to recipe details.
    - If the user is the owner of the recipe, should allow user to delete it.
- Public recipe list:
    - Should display a list of all public recipes ordered by newest as in the wireframe.
    - Should lead to recipe details.
- Recipe details:
    - If the recipe is public or the user is the owner of the recipe, should display the recipe details as in the wireframe.
    - If the user is the owner of the recipe, should lead to the form that allows user to add new food.
- General shopping list view (**do not implement if in group of 3**):
    - Should show the list of food that is missing for all recipes of the logged-in user.
    - Should count the total food items and total price of the missing food.
- Make sure there are no N+1 queries happening.
- Create navigation menu that allows users to open all of the pages you created.

recipe_login

recipe_registration

recipe_food

recipe_recipe

recipe_show

recipe_public

## Thank you for reviewing our project 👍

### CODE REVIEW

**Review by enaburekhan — COMMENTED** [review]
Hi yahyaelganynil, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project with no readme file. Kindly use this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) to create your own readme file. Also ensure the PR passes linters checks.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Code review by **Eric Enaburekhan**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+Feel free to check the [issues page](https://github.com/Mhdez221993/blog_app/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- To my coding partner who helped me through all the work
+- To my Stand Up Team who kept my morale up!
```
Comment (DeliceLydia) [inline README.md:55]: - [ ] Add a live demo in your README

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Major highlights for your project:
- Kudos to the good UI design
- App is working as expected
- No linter errors


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: README.md (hunk 1 above)
Comment (Mhdez221993) [reply README.md]: Done! 👍 
Was deployed to Heroku

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**